src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -230,71 +231,36 @@ private bool TryBindUnconvertedBinaryOperatorToDefaultInterpolatedStringHandler(
                 return false;
             }
 
-            bool isConstant = true;
-            var stack = ArrayBuilder<BoundBinaryOperator>.GetInstance();
+            // The constant value is folded as part of creating the unconverted operator. If there is a constant value, then the top-level binary operator
+            // will have one.
+            bool isConstant = binaryOperator.ConstantValue is not null;
```

Comment:
>bool isConstant = binaryOperator.ConstantValue is not null; [](http://example.com/codeflow?start=12&length=59)

Consider moving the declaration to the line right above the first use of the local.

Reply:
I'll actually move the use here, so we don't do unnecessary walks when there's a constant value.